backend/src/server/routes/v1/secret-sharing-router.ts

```diff
@@ -118,7 +117,7 @@ export const registerSecretSharingRouter = async (server: FastifyZodProvider) =>
         ...req.body,
         accessType: SecretSharingAccessType.Anyone
       });
-      return { id: sharedSecret.id };
+      return { id: sharedSecret.id, hashedHex: sharedSecret.hashedHex };
```

Comment:
Note: In a future fuller revamp of secret sharing, I'd like to keep `id` and `hashedHex` in one instead of returning separate params to keep API simple; this would be achieved by removing direct ID reference to the DB secret sharing ID and instead generating a new one with higher entropy with more compressed text representation than `hex`.

No action needed for this comment for now.